code/Simulation.py

```diff
@@ -142,9 +162,9 @@ def updateLocal(self, delta_t: float):
             s = max(0.000000001, s)
             other_v = carFrontNow.v if carFrontNow is not None else self.v
             self.__v = self.v
-            self.__accel = self.driver.getAccel(v=self.v, other_v=other_v, s=s) * delta_t
+            self.__accel = self.driver.getAccel(v=self.v, other_v=other_v, s=s) * delta_t if not self.failing else 0
```

Comment:
I don't think you need this because self.__accel will be ignored anyways

```suggestion
            self.__accel = self.driver.getAccel(v=self.v, other_v=other_v, s=s) * delta_t
```

Reply:
i do this because we use the other cars accelerations and i wanted to set it to 0